test/string-template.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env mocha */
+'use strict';
+
+const should = require('should');
+const StringTemplate = require('../lib/string-template');
+
+describe('StringTemplate', () => {
+  let string = 'The {{ jumper:adjective }} {{ jumper:color }} {{ jumper:animal }}';
```

Comment:
We might want to change one of these to account for the optional spaces in the capture group e.g. "{{jumper:adjective}}" instead of "{{ jumper:adjective }}".